docs/DeveloperGuide.md

```diff
@@ -143,26 +143,29 @@ How the parsing works:
 
 **API** : [`Model.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/model/Model.java)
 
-<img src="images/ModelClassDiagram.png" width="450" />
+<img src="images/ModelClassDiagramNew.png" width="450" />
```

Comment:
The black diamond for concertContact and person/ concert is wrong since deleting a concert contact does not delete the person/ concert, however the reverse is true and deleting a person/ concert deletes the concert contact if any

Reply:
concertContact is an association class